### ISSUE TITLE
Delete books

### ISSUE THREAD

**Issue by tastytea** [bookwyrm-social/bookwyrm#968]
**Is your feature request related to a problem? Please describe.**
I accidentally created [a duplicate](https://bookwyrm.social/book/122207) and can't delete it.

**Describe the solution you'd like**
I'd like to be able to delete the book. Maybe deletions should require a reason and the admins of the instance have to approve them to prevent malicious deletions?

**Comment by seniorm0ment** [comment]
Maybe a report feature could be beneficial for issues like this, and it sends a report to the instance host.

**Comment by Siinamon** [comment]
I would love to be able to do this on my own instance. Deleting and/or merging books would be incredibly beneficial, especially so that reviews on a given book would be in a single location instead of n+1 "different" books based on federation.

**Comment by jaschaurbach** [comment]
I just was searching how I could delete a book or merge books and came here. So yes, this is needed.

**Comment by fleurc** [comment]
I agree with the above, i just searched Warrior Cats Forest of Secrets, found nothing, imported an Open Library book and fixed its data so that it would at the very least have the actual original author. Found out there are 22 other "editions" which are people doing exactly what i did and then simply having 10 of the same book because of not being able to delete or merge books.

**Comment by iamjmulligan** [comment]
Yeah. Somehow when creating a book today it was made twice so yeah a way to delete would be great but merging dupes would be super handy too

**Comment by oschettler** [comment]
I've just created a duplicate and would like to delete it, so +1

**Comment by fofwisdom** [comment]
What a disaster...

**Comment by Clover-Zero** [comment]
There still isn't a way to delete or merge books after all of these years??
I'm requesting this feature as well. I just found out that there are 2 entries of the same book somehow. I also found 2 entries of the same author, so if possible, I'd like the same feature for authors too.
I agree that the best way might be to report to the admin, or have delete/merge request sent to admin. If the admin accepts, it gets deleted/merged.

**Comment by fofwisdom** [comment]
https://docs.joinbookwyrm.com/management-commands.html

**Comment by mouse-reeve** [comment]
Merging books is actively in development and should be available soon! In the mean time, I strongly discourage using that management command, as it has unintended side effects

**Comment by Clover-Zero** [comment]
Glad to hear that! Thank you!! I'm not the one managing the instance I'm in anyway, I doubt I can use that command.

**Comment by naught101** [comment]
#2664 
#1119
